toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -149,6 +151,18 @@ func main() {
 		logger.Log.Fatalf("Invalid signature handling encountered: %s. Allowed: %s", *signatureHandling, validSignatureLevels)
 	}
 
+	// Setup remote source caching
+	if *sourceCacheDir != "" {
+		fileInfo, err := os.Stat(*sourceCacheDir)
+		if err != nil {
+			logger.Log.Fatalf("Received error getting information for source cache directory. Error: %v", err)
+		}
+		if !fileInfo.IsDir() {
+			logger.Log.Fatalf("Source cache directory (%s) is not a directory", *sourceCacheDir)
+		}
+		templateSrcConfig.localSourceCacheDir = *sourceCacheDir
+	}
```

Comment:
Wouldn't `file.IsDir(*sourceCacheDir)` be enough here?

```suggestion
	templateSrcConfig.localSourceCacheDir = *sourceCacheDir
	if *sourceCacheDir != "" && !file.IsDir(*sourceCacheDir) {
		logger.Log.Fatalf("Source cache directory (%s) is not a directory", *sourceCacheDir)
	}
```

Reply:
Thanks, took the suggestion patch- looks like I was reinventing the wheel there!